views/schools/forms/add-school.hbs

```diff
@@ -138,3 +138,17 @@
     </div>
 {{/if}}
 
+{{#if ENABLE_LDAP_SYNC_DURING_MIGRATION}}<!-- TODO: extend "if" with && schoolIsInMigration -->
```

Comment:
There is an open todo. What says the requirement?

Reply:
thanks for noticing. The requirements have changed in the meantime and the todo is obsolete. I am going to remove this comment